OpenRA.Mods.Common/Traits/World/PaletteFromFile.cs

```diff
@@ -15,6 +15,7 @@
 
 namespace OpenRA.Mods.Common.Traits
 {
+	[Desc("Load Westwood .pal files.")]
```

Comment:
This format isn't Westwood specific.  It directly maps to the VGA palette registers.

Reply:
Renamed.